src/lib.rs

```diff
@@ -9,12 +9,13 @@ pub use prost::{DecodeError, EncodeError, Message};
 #[cfg(feature = "services")]
 pub mod services {
     use crate::{
-        BlockchainTokenTypeV1, BlockchainTxn, DataRate, GatewayStakingMode, MapperAttach, Region,
-        RoutingAddress,
+        BlockchainRegionParamsV1, BlockchainTokenTypeV1, BlockchainTxn, DataRate,
+        GatewayStakingMode, MapperAttach, Region, RoutingAddress,
     };
 
     pub mod iot_config {
         include!(concat!(env!("OUT_DIR"), "/helium.iot_config.rs"));
+        pub use gateway_server::{Gateway, GatewayServer};
```

Comment:
What is GatewayServer here? it's not something any of the iot oracles run right?

Reply:
ah ok yeah, this is fine to keep consistent